docker/compose/docker-compose-nfs.yml

```diff
@@ -0,0 +1,91 @@
+#
```

Comment:
`docker-compose-nfs.yml` represents a way of mounting the NFS share through standard docker. `docker-compose-nfs-mount.yml` gives an example of mounting the NFS share within a docker container. We are following the latter approach with the platform.